repository/favorite.go

```diff
@@ -0,0 +1,12 @@
+package repository
+
+import (
+	"github.com/a-omori-yumemi/YumetterAPI/model"
+)
+
+type IFavoriteRepositoty interface {
```

Comment:
Go でも I 接頭辞書く人いるの驚きました， .NET 書いていたので懐かしいです

```suggestion
type IFavoriteRepository interface {
```